main.py

```diff
@@ -83,19 +83,18 @@ async def IsMemberWorking(ctx, discorduser, date = datetime.today().strftime('%d
 async def IsEveryoneHere(ctx):
 
     guild = client.guilds[0]
-    voice_channel = client.get_channel(int(os.getenv("MeetingChannel_id")))
-    fulltime_role = discord.utils.get(guild.roles, name = "Full Time")
-    fulltime_member_list = list(filter(lambda user: fulltime_role in user.roles, guild.members))
-    fulltime_member_list_in_voicechannel = list(filter(lambda user : fulltime_role in user.roles, voice_channel.members))
+    meeting_channel = client.get_channel(int(os.getenv("MeetingChannel_id")))
+    isAdmin = True if Utilities.IsAdmin(ctx.author) else False
```

Comment:
```suggestion
    isAdmin = Utilities.IsAdmin(ctx.author)
```

True if true, false if false? Redundant. Probably won't then need the variable at all here as well.

Reply:
Done